app/index.html

```diff
@@ -15,6 +15,15 @@
     <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,300,700' rel='stylesheet' type='text/css'>
 
     {{content-for "head-footer"}}
+    <script type="text/javascript"
+        src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.6.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
```

Comment:
It seems that this URL configures it to load TeX, LaTeX, and MathML- very versatile, but also larger file size and possibly slower to render.  

Requirements question: What kind of content, exactly, are we rendering? Do we need to support multiple different notation systems for mathematical content, or can we perhaps ask MathJAX CDN for a more limited subset of its functionality?

http://docs.mathjax.org/en/latest/configuration.html#configuring-mathjax-after-it-is-loaded